pkg/config/config.go

```diff
@@ -233,6 +234,13 @@ type RuntimeConfig struct {
 	// DropInfraCtr determines whether the infra container is dropped when appropriate.
 	DropInfraCtr bool `toml:"drop_infra_ctr"`
 
+	//TODO Beter place for these?
```

Comment:
I'm not very familiar with CRIO's config format so please suggest better names and/or location for these values.